.golangci.yml

```diff
@@ -0,0 +1,138 @@
+# options for analysis running
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 1m
+
+output:
+  sort-results: true
+
+# Uncomment and add a path if needed to exclude
+# skip-dirs:
+#   - some/path
+# skip-files:
+#   - ".*\\.my\\.go$"
+#   - lib/bad.go
+
+# Find the whole list here https://golangci-lint.run/usage/linters/
+linters:
+  disable-all: true
+  enable:
+    - deadcode # finds unused code
+    - errcheck # checking for unchecked errors in go programs
+    - errorlint # errorlint is a linter for that can be used to find code that will cause problems with the error wrapping scheme introduced in Go 1.13.
+    - dupl # tool for code clone detection
+    - forbidigo # forbids identifiers	by reg exps
```

Comment:
```suggestion
    - forbidigo # forbids identifiers used by reg exps
```

Reply:
I disagree with this, `forbidigo` is using regular expressions to forbid certain identifiers. It's rather `forbids identifiers matched by reg exps` but I think in its initial form it's clear enough.

Reply:
Fixed in bc2c107eb5ca81333cf5f5d549143aa7f7e8987f